src/index.html

```diff
@@ -16,7 +19,11 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="page__body">
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__seconds"></div>
+      <div class="stopwatch__minutes"></div>
+      <div class="stopwatch__center"></div>
```

Comment:
Ensure that you have `alt` attributes for accessibility reasons. If these div elements are going to have backgrounds that are images, consider using `img` tags with `alt` attributes instead.